tpm2_pytss/types.py

```diff
@@ -1460,15 +1463,15 @@ class TPM2B_ATTEST(TPM2B_OBJECT):
     pass
 
 
-class TPM2B_CONTEXT_DATA(TPM2B_OBJECT):
+class TPM2B_CONTEXT_DATA(TPM_OBJECT):
```

Comment:
Why are we no longer inheriting? These are TPM2B structures.

Reply:
Seems I looked at the wrong structure, fixed it now.